airbyte-integrations/connectors/source-my-hours/acceptance-test-config.yml

```diff
@@ -14,14 +17,25 @@ acceptance_tests:
   discovery:
     tests:
       - config_path: "secrets/config.json"
+        backward_compatibility_tests_config:
```

Comment:
why is there a breaking change? The schema files look the same to me, except for the explicit null types and some field reordering